CONTRIBUTOR.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to BOM Squad
+
+We welcome contributions to the BOM Squad project! By participating in this project, you agree to abide by the [Code of Conduct](code-of-conduct.md).
+
+## Getting Started
+
+To contribute to the BOM Squad repository, follow these steps:
+
+1. Fork the repository.
+2. Create a new branch for your contribution:
+   ```
+   git checkout -b feature/new-feature
+   ```
+3. Make your desired changes to the codebase.
+4. Commit your changes with a descriptive commit message.
+5. Push your branch to your forked repository:
+   ```
+   git push origin feature/new-feature
+   ```
+6. Open a pull request against the `main` branch of the BOM Squad repository.
+
+
+## Development Guide
+
+For detailed information on setting up your development environment and contributing to the BOM Squad project, please refer to the [Development Guide](/docs/development.md).
```

Comment:
Are there development docs yet? If not, perhaps just add the word "forthcoming."

Reply:
There is a developer.md file, though I am sure it could use some updating. 
```
# Development

### Development Container

1. Docker / Podman
2. VSCode (or other supported IDE)
3. Remote-Containers [extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers)

### Native

1. Go 1.20+
2. Make

## Getting Started

First, clone this repository and make sure you have your enviornment all setup. </br>
For support IDEs users, you can simply run this project in a container (aka devcontainer): </br>

The option would be available once you install the recommended extension. </br>

See more details [here](https://containers.dev/supporting)
```